doc/subsystems/logging/logger.rst

```diff
@@ -109,6 +109,13 @@ be processed by printk. Longer strings are trimmed.
 the log macro call. Note that it can lead to errors when logger is used in the
 interrupt context.
 
+:option:`CONFIG_LOG_PROCESS_TRIGGER_THRESHOLD`: When number of buffered log
+messages reaches the threshold dedicated thread (see :cpp:func:`log_thread_set`)
+is waken up.
```

Comment:
IMHO it might be worth to mention, that this affects internal thread if it is enabled.